akka-cluster/src/main/scala/akka/cluster/ClusterEvent.scala

```diff
@@ -277,6 +282,16 @@ object ClusterEvent {
   /**
    * INTERNAL API
    */
+  private[cluster] def diffUnreachableTeam(oldGossip: Gossip, newGossip: Gossip, selfUniqueAddress: UniqueAddress): immutable.Seq[UnreachableTeam] = {
+    diffUnreachable(oldGossip, newGossip, selfUniqueAddress)
```

Comment:
I don't think this is exactly what we want. It should be based on the observations from `selfTeam` to another team. No other observations should be included, e.g. observations within the other team should not be included, observations from a third team should not be included